lms/djangoapps/instructor/tests/test_tools.py

```diff
@@ -388,11 +388,25 @@ def test_regular(self):
 2014-10-16,10,0,0"""
         self.assertEquals(output, to_match)
 
+
+    def test_edge_case(self):
+        """
+        Test Generating d3 usable csvs
+        """
+        data_string = """ Date,Activity Type,Number New,Votes
+2014-10-8,CommentThread,2,3"""
+        memfile = StringIO.StringIO(data_string)
+        output = tools.generate_course_forums_d3(memfile)
+        to_match = """Date,New Threads,Responses,Comments
+2014-10-8,2,0,0"""
+        self.assertEquals(output, to_match)
+
+
     def test_no_data(self):
         """
         Test Generating d3 usable csvs
         """
-        data_string = "Date,Type,Number,Up Votes,Down Votes,Net Points"
+        data_string = "Date,Type,Number,Votes"
```

Comment:
Perhaps, you want to change the column name 'Number', to 'Number New', to match the naming scheme above?